lessons/07-html-forms/index.html

```diff
@@ -0,0 +1,63 @@
+<html>
+    <head>
+        <title>
+            Create User
+        </title>
+    </head>
+    <body>
+        <main>
+            <h1>Create User</h1>
+            <hr>
+            <form action="create-user.html" method="GET">
+                <label>Email:</label>
+                <input type="email" name="email" placeholder="Email" required/>
+                <br>
+
+                <label>UserName:</label>
+                <input type="text" name="username" placeholder="username" size="30" maxlength="12"/>
+                <br>
+                
+                <label>Password:</label>
+                <input type="text" name="password" placeholder="password" size="30" maxlength="8" required/>
+                <br>
+                
+                <label>Bio:</label><br>
+                <textarea name="bio" placeholder="give a brief description of ths user"></textarea>
+             </form>
+             
+         <form>
+             <fieldset>
+                  <legend>Interests</legend>
+                  <label>Interests:</label>
+                  <form>
+                    Select your favourite superhero
+                    <input type="radio" name="superhero" value="Superman" />Superman 
+                    <input type="radio" name="superhero" value="Batman" /> Batman
+                    <input type="radio" name="superhero" value="Wonder Woman" checked /> Wonder Woman
+                    <input type="radio" name="superhero" value="Flash" />Flash
+                    <input type="radio" name="superhero" value="Thor" /> Thor
+                    <input type="radio" name="superhero" value="Hulk" /> Hulk
+                  </form>
+
+                  <form>
+                    Select your favourite video game
+                    <input type="checkbox" name="videogames" value="Fifa" checked />
+                    Fifa
+                    <input type="checkbox" name="videogames" value="NHL" />
+                    NHL
+                    <input type="checkbox" name="videogames" value="Resident Evil" />
+                    Resident Evil
+                    <input type="checkbox" name="videogames" value="Uncharted" checked />
+                    Uncharted
+                   </form>
+              </fieldset>
+              
+            </form> 
+            <form> 
+               <input type="hidden" id="userid" name="userid" value="2300" />
+               <input type="submit" value="Create User" />
+            </form>
```

Comment:
We don't want these to be inside their own `<form>` tags, we want them to be inside the main `<form action="create-user.html ...>` tag above